main/main.cpp

```diff
@@ -3118,7 +3118,7 @@ Error Main::setup2(bool p_show_boot_logo) {
 		// rendering_driver now held in static global String in main and initialized in setup()
 		Error err;
 		display_server = DisplayServer::create(display_driver_idx, rendering_driver, window_mode, window_vsync_mode, window_flags, window_position, window_size, init_screen, context, init_embed_parent_window_id, err);
-		if (err != OK || display_server == nullptr) {
+		if (display_server == nullptr) {
```

Comment:
I think we should leave these checks as is, just as an additional safeguard to badly implemented versions of `DsiplayServer::create`.

```suggestion
		if (err != OK || display_server == nullptr) {
```

Reply:
yea I agree it wouldn't hurt.

Reply:
Wait, `DisplayServer::create` can't be overridden, can it? If that's the case, the latest changes should do it and we can resolve these discussions